types/maxmsp/index.d.ts

```diff
@@ -567,6 +567,8 @@ declare class Global {
      * TODO: Can have any property assigned to it
      */
     sendnamed(receive_name: string, property_name: string): void;
+
+    [index: string | number | symbol]: unknown;
```

Comment:
We could use `any` instead o `unkown` as it has been done in many other places in this definitions, but I tend to think any is too loose.

Reply:
isn't this more of a pain when we need to get the value though, as you need to then explicitly cast it, eg to a `string`, which you don't need with `any`

Reply:
I changed my mind, I updated the tests, I still believe the indexer approach is better than having to cast global to `any` before it can be used (in that case one can use ts-ignore which would have the same effect).